js/constants/constants.js

```diff
@@ -135,6 +135,21 @@ module.exports = {
     OBJECT: "object"
   },
 
+  TOOLTIPS: {
+    PLAY: "Play",
+    PAUSE: "Pause",
+    REPLAY: "Replay",
+    MUTE: "Mute",
+    UNMUTE: "Unmute",
+    SHARE: "Share",
+    DISCOVER: "Discover",
+    CC: "Closed Captions",
+    VQ: "Video Quality",
```

Comment:
Can we use the full names here such as `VIDEO_QUALITY`? I think it will make it easier to understand when referencing outside this file.